skipper.go

```diff
@@ -349,8 +349,22 @@ type Options struct {
 	// OpenTracing enables opentracing
 	OpenTracing []string
 
-	// PluginDir defines the dir to load plugins from
-	PluginDir string
```

Comment:
let it and mark as deprecated. Use append(PluginDirs, PluginDir) then this will not be a breaking change and we can do later the get rid of deprecated stuff.